config/passport.js

```diff
@@ -24,31 +24,35 @@ const opts = {
 };
 
 module.exports = (passport) => {
-    passport.use(new JwtStrategy(opts, (jwtPayload, done) => {
+    const jwtStrategy = new JwtStrategy(opts, (jwtPayload, done) => {
         User.findOne({ where: { username: jwtPayload.username } })
             .then(user => done(null, user))
             .catch(err => done(err, false));
-    }));
+    });
+
+    const fbStrategy = new FacebookStrategy({
```

Comment:
The FacebookStrategy constructor fails if the environment variables are not set, preventing the service from starting. If facebook config is optional, we should move the constructor back into the `if`